frontend/src/components/DocumentViewer/DocumentViewBox.test.tsx

```diff
@@ -50,7 +49,7 @@ describe('DocumentViewBox component tests', () => {
 
 	const mockDocumentViewer = vi.hoisted(() => vi.fn());
 	vi.mock('@cyntler/react-doc-viewer', () => ({
-		default: (props: DocViewerProps) => {
```

Comment:
DocViewerProps is no longer available as an export in the latest release, but in any case, in the cloud feature branch I've changed the test to render the actual DocViewer component, instead of mocking it, so soon enough this code will be removed.